cdc/scheduler/internal/tp/scheduler_rebalance.go

```diff
@@ -93,8 +93,8 @@ func newBurstBalanceMoveTables(
 	}
 
 	for tableID, rep := range replications {
```

Comment:
it looks that `newBurstBalanceMoveTables` , the parameter `currentTables` is not used.